server/training/train_mlp.py

```
@@ -0,0 +1,141 @@
+                frames = source["frames"]
+                samples = process_clip_to_samples(frames, label, {})
+                all_samples.extend(samples)
+
+    if not all_samples:
+        LOGGER.error("No samples found to train on.")
+        return
+
+    LOGGER.info(f"Generated {len(all_samples)} samples (temporal windows).")
+
+    # 3. Prepare Arrays
+    X, y, labels, weights = dataset_to_arrays(all_samples)
```

Ruff (0.14.10)

Unpacked variable `weights` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)